docs/sql-performance-tuning.md

```diff
@@ -222,17 +222,19 @@ parameter. The "REPARTITION" hint has a partition number, columns, or both/neith
 The "REPARTITION_BY_RANGE" hint must have column names and a partition number is optional. The "REBALANCE"
 hint has an initial partition number, columns, or both/neither of them as parameters.
 
-    SELECT /*+ COALESCE(3) */ * FROM t
-    SELECT /*+ REPARTITION(3) */ * FROM t
-    SELECT /*+ REPARTITION(c) */ * FROM t
-    SELECT /*+ REPARTITION(3, c) */ * FROM t
-    SELECT /*+ REPARTITION */ * FROM t
-    SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
-    SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
-    SELECT /*+ REBALANCE */ * FROM t
-    SELECT /*+ REBALANCE(3) */ * FROM t
-    SELECT /*+ REBALANCE(c) */ * FROM t
-    SELECT /*+ REBALANCE(3, c) */ * FROM t
+```sql
```

Comment:
how about use `{% highlight sql %}` like other SQL highlights in this doc

Reply:
Thanks. I think no need use `{% highlight sql %}`.
* SQL Code block as follow
https://github.com/apache/spark/blob/c480c66d06af9076b9c4af79e51526a054670a06/docs/sql-ref-ansi-compliance.md?plain=1#L45-L67

* Document Page built as follow
![image](https://user-images.githubusercontent.com/44011673/172983757-a8cb8e75-8918-4a4d-9b2f-67fb3df48c68.png)